deploy/zookeeper/quick-start-persistent-volume/zookeeper-3-nodes.yaml

```diff
@@ -90,11 +90,14 @@ spec:
       containers:
         - name: kubernetes-zookeeper
           imagePullPolicy: IfNotPresent
-          image: "docker.io/zookeeper:3.6.1"
+          image: "docker.io/zookeeper:3.7.0"
```

Comment:
Should be 3.6.3

Reply:
fixed on https://github.com/Altinity/acm/commit/9455cf6e87a95de7c0ff36c221d3fe88988c1f79